rhel/SPECS/victoriametrics.spec

```diff
@@ -0,0 +1,59 @@
+%define debug_package %{nil}
+
+%define copying() \
+%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
+%license %{*} \
+%else \
+%doc %{*} \
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         VictoriaMetrics
+%global repo            VictoriaMetrics
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          764b3d4fda41e27aa7af83e406503c1087c41d4d
```

Comment:
@Sudokamikaze @EvgeniyPatlan how that line will work with out `sed`? This repository is a submodule too, so I guess that will not work?

Reply:
@AlekSi it would work like a charm

Reply:
Ok, how and why?

Reply:
@EvgeniyPatlan , are there any updates on this?

Reply:
@EvgeniyPatlan @Sudokamikaze Please answer that

Reply:
`sed -i -e "s/global commit.*/global commit ${full_commit}/" ${rpmbuild_dir}/SOURCES/${spec_name}.spec`
so as you can see `global commit` and everything after it will be replaced with `global commit` and new commit which will be used. full_commit would be obtained from `git -C "${git_dir}" rev-parse HEAD`

Reply:
So that commit should not be there and should be replaces with zeros and a comment, right? And "would work like a charm" means "it will be ignored"?

Reply:
it will be replaced and it does not matter what to put there if sed will be used.
As you asked how it will work without sed - it will work and this commit id will be used.

Reply:
Heh, fun typo. I intended to write "with our" instead of "without".

We should have zeros there:
```
# the lines below are sed'ed by build-server-rpm script to set a correct version
# see: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
%global commit            0000000000000000000000000000000000000000
```

Reply:
But then I don't get why you approved that PR

Reply:
It looks like the sed command works only if the corresponding submodule exists inside `sources` folder at `pmm-submodules` repo. Otherwise sed commands aren't applied - see `if [ -d "${root_dir}/sources/${repo_name}" ]` condition at https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm#L54 .